### PR TITLE
changes for hideTemp() and camelCase LCD api

### PR DESCRIPTION
**Pull request by moinmoin-sh**
Changes to match updated LCD api
This includes a call to` hideTemp()` as well.

### CODE REVIEW

File: src/OXRS_Rack32.cpp [hunk 1]
```diff
@@ -588,7 +594,7 @@ void OXRS_Rack32::_updateTempSensor(void)
     if (temperature != NAN)
```
Comment (sumnerboy12) [inline src/OXRS_Rack32.cpp:594]: Does this need to change to `isnan(temperature)`?

**Review by sumnerboy12 — APPROVED** [review]
Approved with one question about `isnan`

File: src/OXRS_Rack32.cpp (hunk 1 above)
Comment (moinmoin-sh) [reply src/OXRS_Rack32.cpp:594]: good catch. Sure it has to. I overlooked that.
I'll add this tothe PR

Comment (moinmoin-sh) [reply src/OXRS_Rack32.cpp:594]: Should I now also re-revert this one to get temperature display early at startup?
```
  
  // Set the temp sensor resolution (higher res takes longer for reading)
  _tempSensor.setResolution(MCP9808_MODE);
  _lastTempUpdate = -_tempUpdateMs;
}

```
set ` _lasTempUpdate `

Then this PR would solve  #41 

Or di you have a different plan?

**Comment by sumnerboy12** [comment]
Sounds good to me.

**Pull request merged**